doc/source/user_guide/io.rst

```diff
@@ -110,11 +110,12 @@ names : array-like, default ``None``
   List of column names to use. If file contains no header row, then you should
   explicitly pass ``header=None``. Duplicates in this list will cause
   a ``UserWarning`` to be issued.
-index_col :  int or sequence or ``False``, default ``None``
-  Column to use as the row labels of the ``DataFrame``. If a sequence is given, a
-  MultiIndex is used. If you have a malformed file with delimiters at the end of
-  each line, you might consider ``index_col=False`` to force pandas to *not* use
-  the first column as the index (row names).
+index_col : int/string or sequence of int/string or ``False``, default ``None``
```

Comment:
"int, str, sequence of int / str, or False" would be a slightly better way of wording this IMO.

Reply:
I agree, and have implemented the change.